routing/road_graph_router.cpp

```diff
@@ -229,8 +229,8 @@ IRouter::ResultCode RoadGraphRouter::CalculateRoute(m2::PointD const & startPoin
     ASSERT_GREATER(result.distance, 0., ());
 
     CHECK(m_directionsEngine, ());
-    ReconstructRoute(*m_directionsEngine, *m_roadGraph, nullptr /* trafficColoring */,
-                     delegate, result.path, route);
+    ReconstructRoute(*m_directionsEngine, *m_roadGraph, nullptr, delegate, true /* hasAltitude */,
```

Comment:
Yes, previous version of code didn't generate altitudes in ReconstructRoute() at all. Current version of code at release-73 generates some altitude info anyway. ReconstructRoute() is covered by routing_integration_tests only. So I can propose to add an routing_inegration_test.